src/openms_gui/source/VISUAL/TOPPViewSpectraViewBehavior.cpp

```diff
@@ -324,6 +324,16 @@ namespace OpenMS
           peak1d.setIntensity(cpeak.getIntensity());
           spectrum.push_back(peak1d);
         }
+        
+        // Add at least one data point to the chromatogram, otherwise "addLayer" will fail and a segfault occurs later
```

Comment:
could this also be handled in addLayer or during display? This looks like a workaround which does not solve the underlying problem, i.e. no error checking in later stages (which should be robust against these kind of things if used independenly)